dbMigrations/001-addOwnersToContactsAndOrganizations

```diff
@@ -0,0 +1,9 @@
+mongo leadster --eval
```

Comment:
We should discuss with team about naming of migration file. In my opinion, it should be like `V__0.0.1_addOwnersToContactsAndOrganizations`. Im sure that there must be dots between version numbers.

Reply:
The discussion was successful! The offer was rejected!